package.json

```diff
@@ -1,55 +1,46 @@
 {
-  "author": "Jared Allard <jaredallard@outlook.com>",
   "name": "nexe",
-  "description": "create single executables out of your [node/io].js applications",
+  "description": "Create a single executable out of your Node.js application",
   "license": "MIT",
-  "version": "1.1.2",
+  "version": "2.0.0-beta.1",
   "contributors": [
-    {
-      "name": "Criag Condon",
-      "email": "craig.j.condon@gmail.com",
-      "url": "http://crcn.io/"
-    }
+    "Craig Condon <craig.j.condon@gmail.com> (http://crcn.io)",
+    "Jared Allard <jaredallard@outlook.com>",
+    "Caleb Boyd <caleb.boyd@hotmail.com>"
   ],
   "scripts": {
-    "test": "echo node: $(node -v) && node_modules/.bin/mocha --reporter mocha-circleci-reporter test/test.js"
+    "build": "babel src -d lib",
+    "lint": "standard index.js ./src/**/* --fix",
+    "test": "npm run lint"
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/jaredallard/nexe.git"
+    "url": "git://github.com/nexe/nexe.git"
+  },
+  "files": [
```

Comment:
Aaha, I see, you use this instead of `.npmignore`.   I'm not used to seeing this in projects, but as long as it works the same way...